nmdc_schema/migrators/partials/migrator_from_10_2_0_to_11_0_0/migrator_from_10_2_0_to_11_0_0_part_10.py

```diff
@@ -46,7 +46,18 @@ def upgrade(self):
                                         "MetaproteomicsAnalysis", 
                                         "MagsAnalysis",
                                         "ReadQcAnalysis",
-                                        "DataGeneration"]
+                                        "DataGeneration",
+                                        "LibraryPreparation",
+                                        "Pooling",
+                                        "MetagenomeAnnotation",
+                                        "MetagenomeAssembly",
+                                        "MetagenomeSequencing",
+                                        "NomAnalysis",
+                                        "ReadBasedTaxonomyAnalysis",
+                                        "MetatranscriptomeAnnotation",
+                                        "MetatranscriptomeAssembly",
+                                        "MetatranscriptomeExpressionAnalysis"
+                                        ]
```

Comment:
This suggestion goes along with another one.

```suggestion
                                        "MetatranscriptomeExpressionAnalysis",
                                        "CollectingBiosamplesFromSite"
                                        ]
```